src/IconToggle/IconToggle.react.js

```diff
@@ -33,7 +33,7 @@ const propTypes = {
     /**
     * Name of icon to show
     */
-    name: PropTypes.string.isRequired,
+    name: PropTypes.string,
```

Comment:
Yup! That looks like it. Don't forget the `size` prop as well. Any reason why you were doing it like this before though?